changelog_entries/wilderlands_lag.md

```diff
@@ -0,0 +1,3 @@
+### Multiplayer
+   * 5p - The Wilderlands:
+    * Fixed lag during AI turn (issue #10419)
```

Comment:
```suggestion
     * Fixed lag during AI turn (issue #10419)
```

Reply:
Thanks - I cut & pasted from an older changelog entry that used that spacing